src/components/todos/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Todo } from '../../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  selected: string;
+  setStatus: (status: string) => void;
+  removeCompleted: (completedTodos: Todo[]) => void;
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  todos,
+  selected,
+  setStatus,
+  removeCompleted,
+}) => {
+  const statuses = ['All', 'Active', 'Completed'];
```

Comment:
Better use enum here, and in the `App.tsx` too.

It  would be always ➕  in your test tasks in future.